app-devel/mimalloc/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=mimalloc
+PKGDES="A compact general purpose allocator with excellent performance"
+PKGSEC=devel
+PKGDEP="glibc"
+
+CMAKE_AFTER="-DMI_INSTALL_TOPLEVEL=ON"
```

Comment:
Please add a comment explaining that this will be default upstream behaviour (upstream marks the "OFF" setting as deprecated).